apps/workflow_editor/views/App.js

```diff
@@ -6,35 +6,24 @@ import React, { Component } from 'react';
 
 import AppContainer from 'common-web-ui/views/AppContainer';
 
+import { navigation } from '../config/routes';
+
 export default class App extends Component {
 
   state = {
-    title: this.title
+    navigation
   };
 
   render() {
+    document.body.style.overflow = 'hidden';
     return (
       <AppContainer
           ref="container"
-          className="app"
-          header={null}
-          title={this.state.title}
-          children={this.props.children}
-          styles={{content: {padding: 0}}} />
+          disableAppBar={true}
```

Comment:
Class properties must be methods. Expected '(' but instead saw '='.<br>Expected '(' and instead saw '}'.